packages/arb-bridge-peripherals/contracts/tokenbridge/libraries/DaiToken.sol

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.6.11;
+
+/**
+ * @dev Dai Stablecoin ERC20 Contract
+ *
+ * For use when calling permit on tokens that implement permit similarly to Dai.
+ *
+ * https://github.com/makerdao/dss/blob/17187f7d47be2f4c71d218785e1155474bbafe8a/src/dai.sol
+ */
+
+abstract contract DaiToken {
```

Comment:
would prefer `IDaiLikePermit` and only keep the permit interface instead of an abstract contract